cli/cdsctl/admin.go

```diff
@@ -19,6 +19,8 @@ func admin() *cobra.Command {
 			[]*cobra.Command{
 				adminServices,
 				adminHooks,
+				adminRepositories,
```

Comment:
i think i would prefer a common `cdsctl admin services status [--name a-name] [--type a-type]`
because for each king of services this is exactly the same code